tests/console/zypper_lifecycle.pm

```diff
@@ -89,15 +95,13 @@ sub run {
     my $testdate_after  = '2620-02-04';
     my $testdate_before = '2620-02-02';
     # backup and create our lifecycle data with known content
-    select_console 'root-console';
```

Comment:
why remove root-console?

Reply:
only one select_console 'root-console' is enough.